src/Mailtrap.php

```diff
@@ -251,28 +277,4 @@ public function seeInEmailHtmlBody($expected)
         $email = $this->fetchLastMessage();
         $this->assertContains($expected, $email['html_body'], 'Email body contains HTML');
     }
-
-    /**
-     * Look for an attachment on the most recent email.
-     *
-     * @param $count
-     */
-    public function seeAttachments($count)
-    {
-        $attachments = $this->fetchAttachmentsOfLastMessage();
-
-        $this->assertEquals($count, count($attachments));
-    }
-
-    /**
-     * Look for an attachment on the most recent email.
-     *
-     * @param $bool
-     */
-    public function seeAnAttachment($bool)
-    {
-        $attachments = $this->fetchAttachmentsOfLastMessage();
-
-        $this->assertEquals($bool, count($attachments) > 0);
-    }
```

Comment:
why?

Reply:
Agreed with @sergeyklay, no need to delete method because they seem related to the feature you are trying to add.